src/plugins/kibana_react/public/validated_range/validated_dual_range.tsx

```diff
@@ -61,8 +86,8 @@ export class ValidatedDualRange extends Component {
       errorMessage,
     });
 
-    if (isValid) {
-      this.props.onChange(value);
+    if (this.props.onChange) {
```

Comment:
This if logic should not be changed. The entire point of this component is to not call `onChange` with invalid values. That way users can interact with the control and invalid changes stay local.

Reply:
I see your point. The TS linting complained about the possibility of `this.props.onChange` being undefined. Let me see if I can find another option here